core/controller/src/main/scala/whisk/core/controller/Meta.scala

```diff
@@ -226,7 +226,7 @@ trait WhiskMetaApi
     private val webRoutePrefix = pathPrefix(routePath / webInvokePath)
 
     /** Allowed verbs. */
-    private lazy val allowedOperations = get | delete | post | put
+    private lazy val allowedOperations = get | delete | post | put | head | options
```

Comment:
Well it just limits our users. If we don't have any particular concerns, why not?